.clusterfuzzlite/parser_fuzzer.cpp

```diff
@@ -0,0 +1,11 @@
+#include "maddy/parser.h"
+#include <string>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+  std::string payload(reinterpret_cast<const char *>(data), size);
```

Comment:
Is the reinterpret cast really needed? does
```suggestion
  std::string payload(data, size);
```
not work?